File: IzvorniKod/Frontend/src/app/models/Parking.ts
import { ParkingSpace } from "./ParkingSpot";

export class Parking {
    parkingId : number = 0;
    ManagerId : number;
    Name : string;
    PricePerHour : number;
    Description: string;
    idParkingImagePath: string = "";
    NumberOfBikePS: number = 0;
    parkingSpaces: ParkingSpace[];
    constructor(parkingId:number, managerId:number, parkingname:string, price:number, description:string, bikePS: number, parkingSpots: ParkingSpace[] = [], idParkingImagePath: string = "") {
        this.parkingId = parkingId;
        this.ManagerId = managerId;
        this.Name = parkingname;
        this.PricePerHour = price;
        this.Description = description;
        this.parkingSpaces = parkingSpots;
        this.NumberOfBikePS = bikePS;
        this.idParkingImagePath = idParkingImagePath;
    }
}

File: IzvorniKod/Backend/SpotPicker/SpotPicker/EFCore/Parking.cs
using SpotPicker.Models;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace SpotPicker.EFCore
{
    [Table("Parking")]
    public class Parking
    {
        [Key]
        public int Id { get; set; }
     
        [ForeignKey("Manager")]
        public int ManagerId { get; set; }
        public string Name { get; set; }
        public string Description {  get; set; }
        public string? idParkingImagePath { get; set; }

        public double PricePerHour {  get; set; }
        public int NumberOfBikePS {  get; set; }
    }
}